.github/workflows/upload-to-pypi.yml

```diff
@@ -73,8 +73,35 @@ jobs:
         with:
           path: wheelhouse/*.whl
 
+  build_aarch64_wheels:
+    name: Build wheels manylinux_aarch64
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [36, 37, 38, 39]
+        include:
+          - os: ubuntu-latest
+            arch: aarch64
+            platform_id: manylinux_aarch64
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set up QEMU
```

Comment:
What does it do ? It helps with cross-compilation on aarch64 ?

Reply:
It is used for setting up QEMU for enabling AArch64 binary execution on x86 machine.